### PR TITLE
Kusama society rules link update

### LINKED ISSUE #3646: Kusama Guide link to Society rules
https://guide.kusama.network/docs/maintain-guides-society-kusama/#useful-links

Links to https://explorer.polkascan.io/pre/kusama/transaction/0x948d3a4378914341dc7af9220a4c73acb2b3f72a70f14ee8089799da16d94c17

That gives a "Sorry, page not found". I think this is due to Polkascan not allowing access to individual extrinsics via the API anymore, or perhaps some other change. 

Let's change to https://kusama.subscan.io/extrinsic/0x948d3a4378914341dc7af9220a4c73acb2b3f72a70f14ee8089799da16d94c17

Also include instructions for decoding rules (just convert the bytes under the "rules" key to UTF-8)

### PR DESCRIPTION
**Pull request by filippoweb3**
Fixed bug in a link to Kusama society rules

### CODE REVIEW

File: docs/maintain/kusama/maintain-guides-society-kusama.md [hunk 1]
```diff
@@ -187,8 +188,13 @@ society with all their future payouts slashed.
 
 ## Useful links
 
-[Convention of Approval of Membership](https://polkascan.io/pre/kusama/transaction/0x948d3a4378914341dc7af9220a4c73acb2b3f72a70f14ee8089799da16d94c17) -
+[Convention of Approval of Membership](https://kusama.subscan.io/extrinsic/0x948d3a4378914341dc7af9220a4c73acb2b3f72a70f14ee8089799da16d94c17) -
 Rules about joining the Kusama society
 
+In order to decode the rules you need to navigate to the Parameters section: Element 1 ("proposal")
```
Comment (laboon) [inline docs/maintain/kusama/maintain-guides-society-kusama.md]: I would reword this to something like:

"The rules are encoded on-chain in UTF-8 format. This is displayed in most block explorers as a hexadecimal string. In order to see the rules in human-readable format, you can convert it. View the value Element 1.... etc. of the extrinsic's parameters, and copy the value corresponding to the key "Value". You can use a hex-to-UTF8 converter to then display the text. Note that the text is formatted with Markdown."

File: docs/maintain/kusama/maintain-guides-society-kusama.md [hunk 2]
```diff
@@ -187,8 +188,13 @@ society with all their future payouts slashed.
...
+-> "value" -> "params" -> Element 2 ("rules") -> copy the section "value" and paste it into a
+hexadecimal to UTF8 converter.
+
 [substrate pallet]: https://paritytech.github.io/substrate/master/pallet_society/index.html
-[kappa rules]: https://polkascan.io/pre/kusama/transaction/0x948d3a4378914341dc7af9220a4c73acb2b3f72a70f14ee8089799da16d94c17
+[kappa rules]:
+  https://polkascan.io/pre/kusama/transaction/0x948d3a4378914341dc7af9220a4c73acb2b3f72a70f14ee8089799da16d94c17
```
Comment (laboon) [inline docs/maintain/kusama/maintain-guides-society-kusama.md]: This link doesn't work either =(

```suggestion
  https://kusama.subscan.io/extrinsic/0x948d3a4378914341dc7af9220a4c73acb2b3f72a70f14ee8089799da16d94c17
```

**Review by laboon — CHANGES_REQUESTED** [review]
minor changes

File: docs/maintain/kusama/maintain-guides-society-kusama.md (hunk 1 above)
Comment (filippoweb3) [reply docs/maintain/kusama/maintain-guides-society-kusama.md]: Added as requested.

**Review by laboon — APPROVED** [review]
lgtm
